### DISCUSSION TITLE
How to install helix in iSH on iOS?

### DISCUSSION THREAD

**Discussion by ylorn** [helix-editor/helix#5178] [Q&A] [answered]
I assume that helix supports Alpine Linux from the installation documentation. Is it included in [APK package manager](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management)? I was trying to install it to [iSH](https://apps.apple.com/us/app/ish-shell/id1436902243) but cannot find it with `apk`.

If it is not included in APK, can we support it?

**Comment by pascalkuthe** [comment, marked as answer]
Helix is included in the [official alpine linux repos ](https://pkgs.alpinelinux.org/package/edge/community/aarch64/helix#) altough the version there is about 4 months out of date (but will probably/hopefully update soon).

However it seems that `ish` [uses an old alpine linux repo](https://github.com/ish-app/ish/wiki/Using-Alpine-Linux-repositories).
They currently use 3.14 which is from mid 2021. For that version helix was not packed yet (and the helix from back then is vastly different from the helix of today anyway). 

The best option for you would probably be to install [rustup](https://rustup.rs/)  (the `rustc` in those old repos is also too old to build helix)  and compile helix yourself as described in the readme.

**Reply by ylorn** [reply]
Thank you! I think I will give rustup a go and try. Also, I will wait for the helix to be updated in alpine linux repo and maybe inform the author of iSH to upgrade the latest repo then.
